source/style-guide/code/js/.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
General question:
Some of these settings are current defaults and do not need to be included. Is the intent to "freeze" these values, and if so, perhaps we should specify the entire options schema?

Reply:
Default values reference (https://prettier.io/docs/en/options.html)